notification-hubs-sdk/build.gradle

```diff
@@ -18,6 +18,9 @@ android {
         versionName VERSION
         testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
         testInstrumentationRunnerArguments clearPackageData: 'true'
+
+        buildConfigField 'int', 'VERSION_CODE', "${versionCode}"
+        buildConfigField 'String', 'VERSION_NAME', "\"${versionName}\""
```

Comment:
Please delete these lines. We can already use gradle provided version information because of the `versionCode` and `versionName` fields above. You can see them already in use here: https://github.com/Azure/azure-notificationhubs-android/blob/b94ad9f8ca6011a37d3560b567d5714288a19264/notification-hubs-sdk/src/main/java/com/microsoft/windowsazure/messaging/notificationhubs/InstallationPutRequest.java#L138-L144